scripts/create-container.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+# Flags
+# set -e
+
+# Local default vars
+name="timeoverflow"
+template="/usr/share/lxc/templates/lxc-ubuntu"
+rls="xenial"
+lxc_config="/tmp/ubuntu.$name.conf"
+host="local.timeoverflow.org"
+project_name="timeoverflow"
+project_path="${PWD%/*}/$project_name"
+app_user="ubuntu"
```

Comment:
Since we source the same defaults from `lxc.cfg` in line `#18`, are these defaults sill needed?